slam_gmapping/src/slam_gmapping.cpp

```diff
@@ -22,9 +22,16 @@
 //
 
 #include "slam_gmapping/slam_gmapping.h"
+using namespace std::chrono_literals;
 
 #define MAP_IDX(sx, i, j) ((sx) * (j) + (i))
 
+#define LOAD_PARAMS(_name_,_val_) \
+   this->declare_parameter((#_name_),(_val_),\
+      rcl_interfaces::msg::ParameterDescriptor());\
+   this->get_parameter((#_name_), (_name_##_));\
+   std::cout<<(#_name_)<<" "<<(_name_##_)<<std::endl
+
```

Comment:
Thank you for the modifications @skylerpan. Some of the parameter related changes are still not reverted. Can you please revert them as well. Please feel free to have them as a separate PR.

Reply:
Hi @shiveshkhaitan. I reverted all the code excepting two bug fixes, and also rebased the commits. Many thanks.